pwndbg/heap/ptmalloc.py

```diff
@@ -449,6 +451,8 @@ def bin_at(self, index, arena_addr=None):
         Bin 1          - Unsorted BiN
         Bin 2 to 63    - Smallbins
         Bin 64 to 126  - Largebins
+
+        Returns: tuple(chain_from_bin_fd, chain_from_bin_bk, is_chain_corrupted) or None
```

Comment:
Does it detect all cases of corrupted chains? What if there is a loop in the linked list pointers (I guess there might be one).
Might be good to add some docs about it

Reply:
i think all cases. Can't get into loop, because `pwndbg.chain.get` stops when loop is detected.
Docs seems unnecesary, as the check is simple.

Reply:
The docs could document examples how those commands behave when heap is corrupted (with examples)